### PR TITLE
Some docstrings/comments, and remove "standalone" wrappers

### PR DESCRIPTION
**Pull request by colinmorris**
As far as I can tell, standalone wasn't being used anywhere, nor was anyone using the default env value of None for any wrappers (here or in gym).

### CODE REVIEW

File: universe/wrappers/vectorize.py [hunk 1]
```diff
@@ -58,6 +56,7 @@ def _seed(self, seed):
 
 class WeakUnvectorize(Unvectorize):
     def __init__(self, env, i):
+        # What is i? Seems not to be used anywhere?
```
Comment (colinmorris) [inline universe/wrappers/vectorize.py]: Anyone want to chime in on this one? There are no obvious usages of this attribute, and removing it doesn't break any tests. Artefact of some earlier capability maybe?

Comment (gdb) [inline universe/wrappers/vectorize.py]: Yeah, seems safe to remove. Probably a debugging thing.

File: universe/vectorized/core.py [hunk 2]
```diff
@@ -3,11 +3,16 @@
...
     metadata = {
-        'runtime.vectorized': True
+        # This key indicates whether an env is vectorized (or, in the case of
+        # Wrappers where autovectorize=True, whether they should automatically
+        # be wrapped by a Vectorize wrapper.)
+        'runtime.vectorized': True,
     }
 
-    # User should set this!
+    # Number of remotes. User should set this (by calling configure())
```
Comment (gdb) [inline universe/vectorized/core.py]: Not always by calling configure; that'll depend on the Env implementation.

**Comment by gdb** [comment]
Cool, ready to merge?

**Comment by colinmorris** [comment]
Sure, I'm happy with it.

**Pull request merged**